docs/index.rst

```diff
@@ -1,3 +1,6 @@
+.. image:: ../src/ert/gui/resources/gui/img/ert_icon.svg
```

Comment:
Should either scale down or I believe the theme has builtin support for icon which should be used instead.

Reply:
The icon support is only for some supported icons I believe?